client/session/session_storage_test.go

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package session
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	stats_dto "github.com/mysteriumnetwork/node/client/stats/dto"
+	"github.com/mysteriumnetwork/node/core/connection"
+	discovery_dto "github.com/mysteriumnetwork/node/service_discovery/dto"
+	"github.com/stretchr/testify/assert"
+
```

Comment:
It this editor formats imports this way with an extra empty line in the middle?

Reply:
It's totally fine with that as it allows for multiple "blocks" of imports, hence if I add an empty line - it will allow for it without complaining :)

Reply:
fixd